project/src/main/resources/ScapegoatInspections.scala

```diff
@@ -15,32 +15,22 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-package com.mwz.sonar.scala
-package scapegoat
+package com.mwz.sonar.scala.metadata.scapegoat
 
-import org.sonar.api.batch.rule.Severity
-
-private[scapegoat] sealed trait Level {
-  def toRuleSeverity: Severity = this match {
-    case Level.Error   => Severity.MAJOR
-    case Level.Warning => Severity.MINOR
-    case Level.Info    => Severity.INFO
-  }
-}
-
-private[scapegoat] object Level {
+private[metadata] sealed trait Level
+object Level {
   case object Error extends Level
   case object Warning extends Level
   case object Info extends Level
```

Comment:
ditto **case objects** should be `final`.

Reply:
thanks for the reminder, I keep forgetting 😅